Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -1135,11 +1135,9 @@ public extension ChatChannelController {
     ///
     /// Please check [documentation](https://getstream.io/chat/docs/android/watch_channel/?language=swift) for more information.
     ///
-    /// We keep these functions internal since we're not sure how we should interface this behavior.
-    /// If you have suggestions, please open a ticket or send us an email at support@getstream.io
     ///
     /// - Parameter completion: Called when the API call is finished. Called with `Error` if the remote update fails.
-    internal func startWatching(completion: ((Error?) -> Void)? = nil) {
+    func startWatching(completion: ((Error?) -> Void)? = nil) {
```

Comment:
Shouldn't these be public?

Reply:
the extension where these funcs are defined is already public